### DISCUSSION TITLE
Is this production ready?

### DISCUSSION THREAD

**Discussion by ShaikhSuf** [drkNsubuga/PharmaSpot#21] [Q&A] [answered]
Hello Guys,

Congratulations on this product. Is this production ready? I want to try it for my new pharmacy. Does this run on Mac? 

Thanks,
-Sufian

**Comment by drkNsubuga** [comment, marked as answer]
Thanks for the appreciation. This is an early version and more features are still in development. You can run a test-drive and see if it'll work for you. Yes, it runs on Mac!
